src/pages/Dashboard/Dashboard.scss

```diff
@@ -10,7 +11,14 @@
   }
 }
 @media screen and (min-width: $pf-global--breakpoint--xs) and (max-width: $pf-global--breakpoint--sm) {
+  .dashboard-carddisplay img:nth-last-of-type(-n + 2) {
+    display: none;
```

Comment:
what is the purpose of these CSS?

Reply:
> what is the purpose of these CSS?

no purpose .i was testing out something and i forgot to delete it.